code/modules/mob/living/carbon/human/human.dm

```diff
@@ -949,10 +949,20 @@
 	return (ishuman(target) && !(target.mobility_flags & MOBILITY_STAND))
 
 /mob/living/carbon/human/proc/fireman_carry(mob/living/carbon/target)
-	if(can_be_firemanned(target))
-		visible_message("<span class='notice'>[src] starts lifting [target] onto their back...</span>",
-			"<span class='notice'>You start lifting [target] onto your back...</span>")
-		if(do_after(src, 50, TRUE, target))
+	var/carrydelay = 50 //if you have latex you are faster at grabbing
+	var/skills_space = " " //cobby told me to do this

```

Comment:
haha whoops

you could also just have this as `""` and then change the message to `visible_message("<span class='notice'>[src] starts [skills_space] lifting [target] onto their back..</span>"`

dunno what I was thinking 😏

Reply:
it is done